modules/kmm-gathering-data-for-kmm-hub.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-gathering-data-for-kmm-hub_{context}"]
+= Gathering data for KMM-Hub
+
+.Procedure
+
+. Gather the data for the KMM operator hub controller manager:
++
+[source,terminal]
+----
+export MUST_GATHER_IMAGE=$(oc get deployment -n openshift-kmm-hub kmm-operator-hub-controller-manager -ojsonpath='{.spec.template.spec.containers[?(@.name=="manager")].env[?(@.name=="RELATED_IMAGES_MUST_GATHER")].value}')
+----
++
+[source,terminal]
+----
+$ oc adm must-gather --image="${MUST_GATHER_IMAGE}" -- /usr/bin/gather -u
+----
++
+[NOTE]
+====
+Use the `-n NAMESPACE` switch to specify a namespace if you installed KMM in a custom namespace.
+====
+
+. View the operator logs:
++
+[source,terminal]
+----
+$ oc logs -fn openshift-kmm-hub deployments/kmm-operator-hub-controller-manager
+----
++
+.Example output
+[source,terminal]
+----
+
```

Comment:
@cdvultur Do we need to document collecting audit logs? Ex: $ oc adm must-gather -- /usr/bin/gather_audit_logs

Reply:
Not sure on the `gather_audit_logs` . Tried to run it on my setup and I get `/usr/bin/gather_audit_logs: No such file or directory`

Maybe @qbarrand can help on this one

```
$ oc adm must-gather --image="${MUST_GATHER_IMAGE}" -- /usr/bin/gather_audit_logs
[must-gather      ] OUT Using must-gather plug-in image: registry.redhat.io/kmm/kernel-module-management-must-gather-rhel8@sha256:4a6b25ba3b4302e940ebbdacfeb6280a008edd56475363cd93087dcfc322d344
When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
ClusterID: 6ad73595-913b-4fae-a52e-ac70b3c98ae0
ClusterVersion: Stable at "4.12.0-0.nightly-2023-02-22-173620"
ClusterOperators:
	All healthy and stable


[must-gather      ] OUT namespace/openshift-must-gather-57g79 created
[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-4th8k created
[must-gather      ] OUT pod for plug-in image registry.redhat.io/kmm/kernel-module-management-must-gather-rhel8@sha256:4a6b25ba3b4302e940ebbdacfeb6280a008edd56475363cd93087dcfc322d344 created
[must-gather-x6tg7] POD 2023-04-17T11:27:22.226561557Z /bin/bash: /usr/bin/gather_audit_logs: No such file or directory
[must-gather-x6tg7] OUT waiting for gather to complete

```

Reply:
Our must-gather image should collect everything we need. Thank you for bringing this up though.